Reneiski_Homework1/Reneiski_Homework1/Lecture5/Zavod.cs

```diff
@@ -0,0 +1,29 @@
+﻿using WorkerNamespace;
+
+namespace Reneiski_Homework1.Lecture5
+{
+    public class Zavod
+    {
+        public Worker? Engineer;
+        public Worker? Building;
+
+        public string Name;
```

Comment:
It's better to make variables below private and put in the beginning of the class:
 public string Name;
        public string Description;
        public int HightOftube;
        public string NameOfbuilding;

Reply:
Fixed, but lastName and position are used in class Zavod so I left them public

Reply:
Fixed. Two public methods were created to get values outside of the class

Reply:
make Name and Description private

Reply:
Fixed! now they are private!